### PR TITLE
Improve wording of the unauthenticated media deprecation box

### PR DESCRIPTION
**Pull request by HarHarLinks**
### Pull Request Checklist

* [x] Pull request includes a [changelog file](https://github.com/matrix-org/matrix-spec/blob/master/CONTRIBUTING.rst#adding-to-the-changelog)
* [x] Pull request includes a [sign off](https://github.com/matrix-org/matrix-spec/blob/master/CONTRIBUTING.rst#sign-off)
* [x] Pull request is classified as ['other changes'](https://github.com/matrix-org/matrix-spec/blob/master/CONTRIBUTING.rst#other-changes)

Signed-off-by: Kim Brose <2803622+HarHarLinks@users.noreply.github.com>

Preview: https://pr1916--matrix-spec-previews.netlify.app

### CODE REVIEW

File: content/client-server-api/modules/content_repo.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ mxc://<server-name>/<media-id>
 Clients can access the content repository using the following endpoints.
 
 {{% boxes/added-in-paragraph %}}
-{{< changed-in v="1.11" >}} Clients SHOULD NOT use the deprecated media endpoints
-described below. Instead, they SHOULD use the new endpoints which require authentication.
+{{< changed-in v="1.11" >}} Clients SHOULD NOT use the media endpoints marked as
+deprecated below. Instead, they SHOULD use the new endpoints which require authentication.
```
Comment (Johennes) [inline content/client-server-api/modules/content_repo.md:53]: To echo what was already said in the SCT room: This seems like an improvement over the previous text but it still mainly just rehashes what "deprecated" means. @richvdh's suggestion was something along the lines of:

> "[Changed in v1.11] A number of endpoints under the /_matrix/media hierarchy have been deprecated and replaced with new endpoints which require authentication"

Comment (HarHarLinks) [inline content/client-server-api/modules/content_repo.md:53]:
```suggestion
{{< changed-in v="1.11" >}} A number of endpoints under the /_matrix/media hierarchy
have been deprecated and replaced with new endpoints which require authentication.
The deprecated endpoints are marked in the section below.
```

Comment (HarHarLinks) [reply content/client-server-api/modules/content_repo.md:53]: Rephrase the unauthenticated media deprecation box based on feedback

Comment (HarHarLinks) [reply content/client-server-api/modules/content_repo.md]: Thanks, I've adjusted the text based on this feedback

**Review by richvdh — APPROVED** [review]
thanks!

**Pull request merged**